test/Sema/superfluously-public-imports.swift

```diff
@@ -96,6 +97,9 @@ public struct NonPublicExtendedType {}
 //--- UnusedImport.swift
 
 //--- UnusedPackageImport.swift
```

Comment:
Is it intentional for `UnusedPackageImport.swift` to be an empty file?

Reply:
Yes, it's imported but nothing it would contain would be used, but the import does show the diagnostic that we're testing. I could have had a single Empty.swift file and compile it into many different modules, but I find it useful to have a file with the module name.